app/src/hooks/useKeycloakWrapper.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { cleanup } from '@testing-library/react';
```

Comment:
what is this file for.... seems incomplete and possibly in the wrong spot

Reply:
Not sure haha, will remove